sklearn/linear_model/_coordinate_descent.py

```diff
@@ -452,7 +451,7 @@ def enet_path(X, y, *, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
     multi_output = False
     if y.ndim != 1:
         multi_output = True
-        _, n_outputs = y.shape
+        n_targets = y.shape[1]
```

Comment:
If one wants to be explicit, I think this is clearer:
```suggestion
        _, n_targets = y.shape
```

Reply:
With the unpacking, we get more context and we explicitly know that there are two values solely.

But it's really a minor suggestion/nitpick.